app/src/main/java/com/example/parkingsystem_kotlin/mvp/contract/ParkingContract.kt

```diff
@@ -3,13 +3,15 @@ package com.example.parkingsystem_kotlin.mvp.contract
 interface ParkingContract {
     interface Presenter{
         fun onParkingSizeButtonPressed()
+        fun onParkingSizeSet(parkingSize: String)
     }
 
     interface Model{
-        fun getParkingSize(): Int
+        var parkingSize: Int;
```

Comment:
remember that the ; is not necessary in kotlin so please remove it. On the other hand, do not put variables on the contacts, just keet a method to keep the encapsulation

Reply:
done